.github/workflows/integration.yml

```diff
@@ -15,7 +15,7 @@ on:
 
 jobs:
   test:
-    name: WDL Integration Tests (Staging)
+    name: dxCompiler Release Tests (Staging)
```

Comment:
This should rather be dxCompiler Integration Tests (Staging) and github/workflows/release.yml  -> dxCompiler Release(Staging)